tensorflow/compiler/mlir/hlo/tests/lhlo-legalize-to-affine.mlir

```diff
@@ -202,3 +202,39 @@ func @int_dot_op(%lhs: memref<7x3xi32>, %rhs:
     (memref<7x3xi32>, memref<3x4xi32>, memref<7x4xi32>) -> ()
   return
 }
+
+// CHECK-LABEL: func @concatenate
+func @concatenate(%arg0: memref<1x1xf32>, %arg1: memref<1x100xf32>, %arg2: memref<1x200xf32>, %arg3: memref<1x301xf32>) {
+    // CHECK-NEXT:    %[[RESULT:.*]] = alloc() : memref<1x301xf32>
+    // CHECK-NEXT:    affine.for %[[X:.*]] = 0 to 1 {
+    // CHECK-NEXT:      affine.for %[[Y:.*]] = 0 to 1 {
+    // CHECK-NEXT:        %[[LOAD:.*]] = affine.load %arg0[%[[X]], %[[Y]]] : memref<1x1xf32>
+    // CHECK-NEXT:        affine.store %[[LOAD]], %[[RESULT]][%[[X]], %[[Y]]] : memref<1x301xf32>
+    // CHECK-NEXT:      }
+    // CHECK-NEXT:    }
+    // CHECK-NEXT:    affine.for %[[X:.*]] = 0 to 1 {
+    // CHECK-NEXT:      affine.for %[[Y:.*]] = 1 to 101 {
+    // CHECK-NEXT:        %[[LOAD:.*]] = affine.load %arg1[%[[X]], %[[Y]] - 1] : memref<1x100xf32>
+    // CHECK-NEXT:        affine.store %[[LOAD]], %[[RESULT]][%[[X]], %[[Y]]] : memref<1x301xf32>
+    // CHECK-NEXT:      }
+    // CHECK-NEXT:    }
+    // CHECK-NEXT:    affine.for %[[X:.*]] = 0 to 1 {
+    // CHECK-NEXT:      affine.for %[[Y:.*]] = 101 to 301 {
+    // CHECK-NEXT:        %[[LOAD:.*]] = affine.load %arg2[%[[X]], %[[Y]] - 101] : memref<1x200xf32>
+    // CHECK-NEXT:        affine.store %[[LOAD]], %[[RESULT]][%[[X]], %[[Y]]] : memref<1x301xf32>
+    %0 = alloc() : memref<1x301xf32>
```

Comment:
Can you rebase and fix the test? It is likely memref.alloc here I believe now.

Reply:
Done.